controllers/pkg/reconcilers/config/config.go

```diff
@@ -19,12 +19,15 @@ package ctrlrconfig
 import (
 	"time"
 
+	"github.com/henderiw-nephio/nephio-controllers/pkg/giteaclient"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 )
 
 type ControllerConfig struct {
 	PorchClient client.Client
+	GiteaClient giteaclient.GiteaClient
```

Comment:
Is this config object going to contain some sort of entries for each reconciler? We may want to refactor that at some point. But this is OK for now.

Reply:
Not for each reconciler, it will be meant for the repo controller.  
 I will refactor this when integrating repo-controller with controller-manager,  I am not sure if we expect this as env variable for that controller .